gapic-common/lib/gapic/call_options.rb

```diff
@@ -42,7 +42,9 @@ class CallOptions
     #
     def initialize timeout: nil, metadata: nil, retry_policy: nil
       # Converts hash and nil to a policy object
-      retry_policy = RetryPolicy.new retry_policy.to_h if retry_policy.respond_to? :to_h
+      if retry_policy.respond_to? :to_h
+        retry_policy = RetryPolicy.new **retry_policy.to_h
```

Comment:
I suspect rubocop will mark this as ambiguous, and suggest including parens. I also suspect it will ask you to use a modifier conditional since it will fit on one line.
```ruby
retry_policy = RetryPolicy.new(**retry_policy.to_h) if retry_policy.respond_to? :to_h
```